business_objects/monitor.py

```diff
@@ -1,39 +1,31 @@
 from typing import Any, List, Optional
 from . import general
 from .. import enums
-
+from ..models import TaskQueue, Organization
 from ..util import prevent_sql_injection
+from ..session import session
 
 
 def get_all_tasks(
-    project_id: Optional[str] = None,
+    page: int = 1,
+    limit: int = 100,
     only_running: bool = True,
```

Comment:
`only_running` seems to be unused. Can be removed? Or kept for reference for future usecase?